deploy/examples/create-external-cluster-resources.py

```diff
@@ -1440,27 +1454,44 @@ def _gen_output_map(self):
             if self._arg_parser.dry_run:
                 self.create_rgw_admin_ops_user()
             else:
-                err = self.validate_rgw_multisite(
-                    self._arg_parser.rgw_realm_name, "--rgw-realm"
-                )
-                err = self.validate_rgw_multisite(
-                    self._arg_parser.rgw_zonegroup_name, "--rgw-zonegroup"
-                )
-                err = self.validate_rgw_multisite(
-                    self._arg_parser.rgw_zone_name, "--rgw-zone"
-                )
-                (
-                    self.out_map["RGW_ADMIN_OPS_USER_ACCESS_KEY"],
-                    self.out_map["RGW_ADMIN_OPS_USER_SECRET_KEY"],
-                    info_cap_supported,
-                    err,
-                ) = self.create_rgw_admin_ops_user()
-                err = self.validate_rgw_endpoint(info_cap_supported)
-                if self._arg_parser.rgw_tls_cert_path:
-                    self.out_map["RGW_TLS_CERT"] = self.validate_rgw_endpoint_tls_cert()
-                # if there is no error, set the RGW_ENDPOINT
-                if err != "-1":
-                    self.out_map["RGW_ENDPOINT"] = self._arg_parser.rgw_endpoint
+                if not (
```

Comment:
I see the logic works, but it is difficult to read the code with the `not` at the top. Could you change the pattern to be this?
```
if (all blank)
  or
  (all set):
  // set the params
else:
  err = "Please provide all the RGW multisite parameters or none of them"
  sys.stderr.write(err)

Reply:
Updated.